dataFiller/src/main/java/com/inctinctools/marathon2017/socmap/datafiller/model/ElementsList.java

```diff
@@ -0,0 +1,30 @@
+package com.inctinctools.marathon2017.socmap.datafiller.model;
+
+import com.inctinctools.marathon2017.socmap.data.entities.POI;
+
+import java.util.LinkedList;
+import java.util.List;
+
+public class ElementsList {
+    private List<Element> list;
+
+    public ElementsList(){
+        list = new LinkedList<Element>();
```

Comment:
Regarding performance LinkedList vs ArrayList
Linked list a little bit faster then we add something to it
but it much more slower then we getting items
please check:
https://dzone.com/articles/arraylist-vs-linkedlist-vs